src/main/proto/wfa/measurement/api/v2alpha/protocol_config.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.api.v2alpha;
+
+import "wfa/measurement/api/v2alpha/differential_privacy.proto";
+import "wfa/measurement/api/v2alpha/sketch_config.proto";
+
+option java_package = "org.wfanet.measurement.api.v2alpha";
+option java_multiple_files = true;
+option java_outer_classname = "ProtocolConfigProto";
+
+// Resource message for a multi-party computation (MPC) protocol. Immutable.
+message ProtocolConfig {
+  message Key {
+    string protocol_config_id = 1;
+  }
+  // Resource key.
+  Key key = 1;
+
+  // Type of measurement.
+  //
+  // The values should correspond to the fields in the `measurement_type` Oneof
+  // in `MeasurementSpec`.
+  enum MeasurementType {
+    MEASUREMENT_TYPE_UNSPECIFIED = 0;
+    REACH_AND_FREQUENCY = 1;
+  }
+  // The type of measurement that this protocol computes. Required. Immutable.
+  MeasurementType measurement_type = 2;
+
+  // Configuration for the Liquid Legions v2 R/F protocol.
+  message LiquidLegionsV2 {
+    // Parameters for sketch. Required.  Immutable.
+    LiquidLegionsSketchParams sketch_params = 1;
+
+    // Differential privacy parameters for data provider noise registers.
+    // Required.  Immutable.
+    //
+    // Each of these registers contains a well-known constant register ID, and
+    // arbitrary key and count values.
+    DifferentialPrivacyParams data_provider_noise = 2;
+
+    // Sketch configuration, generated from `sketch_params`. Output-only.
+    // Immutable.
+    SketchConfig sketch_config = 3;
```

Comment:
Sorry I didn't raise this up during previous discussion.

I already moved the Sketch/SketchConfig proto into the AnySketch repo.
https://github.com/world-federation-of-advertisers/any-sketch/blob/main/src/main/proto/wfa/any_sketch/sketch.proto

For sure, the API definition shouldn't use proto defined outside, but it feels it is painful to convert a v2alpha.SketchConfig to anysketch.SketchConfig since there are so many distributions inside or maybe even more in the future.

The only uncertain part during converting a LiquidLegionsSketchParams proto to SketchConfig proto is the constant used to mod the hashed key
https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/main/src/main/kotlin/org/wfanet/measurement/loadtest/config/liquid_legions_sketch_config.textproto#L17

In other words, to make sure all EDPs get the same SketchConfig for LLV2, we only need to provide an extra int64 SamplingIndicatorSize in the requisition.

maybe replace this `SketchConfig sketch_config = 3;` with `int64 sampling_indicator_size = 3;` and delete the sketch_config.proto

Reply:
> > _[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/13)_ status: 0 of 8 files reviewed, 4 unresolved discussions (waiting on @kungfucraig, @SanjayVas, and @wangyaopw)
> 
> _[src/main/proto/wfa/measurement/api/v2alpha/protocol_config.proto, line 59 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/13#-MYqwOMtpj4az_R5v15e-r1-59:-MYug6K57VEvrlxHfFIq:b8yquyz) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement-api/blob/9ea3d39fc6ae080e640cda08cccbd5ebf76b142f/src/main/proto/wfa/measurement/api/v2alpha/protocol_config.proto#L59)):_
> 
> Previously, wangyaopw (Yao Wang) wrote…
> I'm not terribly concerned with having to convert between API and external versions of proto messages, as that's sort of par for the course when it comes to having a gRPC API. The concern is mainly about how likely it is that separate EDP client implementations produce incompatible sketches. This includes multiple versions of the same client library.
> 
> If you think that `LiquidLegionsSketchParams` with the additional field is sufficient for all EDPs to generate compatible sketches, then I'm happy to switch to that.
> 
> Note that the API makes no assumptions about whether the client is using any particular library/service, whether it be some turn-key EDP or even AnySketch. It can assume they have access to those implementations as reference though.

Yep, a sketchConfig is no better than a LiquidLegionsSketchParams + the additional field, they contain the exact same information inside.